ui/jobs/buff_tracker.ts

```diff
@@ -314,13 +317,27 @@ export class BuffTracker {
         cooldown: 180,
       },
       embolden: {
-        // Embolden is special and has some extra text at the end, depending on embolden stage:
-        // Potato Chippy gains the effect of Embolden from Tater Tot for 20.00 Seconds. (5)
-        // Instead, use somebody using the effect on you:
-        //   16:106C22EF:Tater Tot:1D60:Embolden:106C22EF:Potato Chippy:500020F:4D7: etc etc
-        gainAbility: kAbility.Embolden,
+        // On each embolden stack changes,
+        // there will be a gain effect log with a wrong duration (always 20).
+        // So using stack to identify the first log.
+        gainEffect: EffectId.Embolden,
         loseEffect: EffectId.Embolden,
         durationSeconds: 20,
+        stack: 5,
+        icon: '../../resources/ffxiv/status/embolden.png',
+        // Lime.
+        borderColor: '#57FC4A',
+        sortKey: 3,
+        cooldown: 120,
+      },
+      emboldenself: {
+        // RDM himself gains a different buff with a same name.
+        // EffectId forbids a name to have two ids,
+        // so hard code here.
```

Comment:
What do you think about calling this `EmboldenSelf` in the effect_id list with a comment about it?

Reply:
Then needs a refactoring in `gen_effect_id.py`, to add a `ManualAbilityMap` or something.

Reply:
Yeah, I think adding the effect name there is better than just having an explicit value here.